segmentstore/server/src/main/java/io/pravega/segmentstore/server/SegmentStoreMetrics.java

```diff
@@ -313,4 +328,17 @@ public void truncate() {
     }
 
     //endregion
+
+    //region RecoveryProcessor
+
+    /**
+     * RecoveryProcessor metrics.
+     *
+     * @param duration Time taken for a Segment Store instance to perform the recovery of containers.
+     */
+    public static void recoveryCompleted(long duration) {
```

Comment:
This should report the recovery per container. Some containers may have longer recovery times since they may have many more operations to recover. For example, in some of our past system tests, 2 containers had a 50,000 operation backlog, while the other 2 had less than 1000. Their recovery times were proportional to this. If we report this as a global metric, the actual durations may get smoothened out (especially if we have a lot of containers) so we won't be able to detect any spikes).